bigquery/tests/system.py

```diff
@@ -385,58 +385,54 @@ def _fetch_single_page(table, selected_fields=None):
         page = six.next(iterator.pages)
         return list(page)
 
-    def _create_table_many_columns(self, rows):
-        # Load a table with many columns
-        dataset = self.temp_dataset(_make_dataset_id('list_rows'))
+    def _create_table_many_columns(self, rowcount):
+        # Generate a table of maximum width via CREATE TABLE AS SELECT.
+        # first column is named 'rowval', and has a value from 1..rowcount
+        # Subsequent column is named col_<N> and contains the value N*rowval,
+        # where N is between 1 and 9999 inclusive.
+        dsname = _make_dataset_id('wide_schema')
```

Comment:
Any reason not to mangle the dataset name using `_make_dataset_id`?  This test will fail is some future condition causes the bare `wide_dataset` dataset not to be deleted.

Reply:
Looks like he is using `_make_dataset_id` to me. Maybe this comment was on an old commit?

Reply:
Yes, _make_dataset_id suffixes the wide_schema prefix to avoid name collisions.

Reply:
Hmm, maybe my eyes are going.